src/actions/sendContactForm.ts

```diff
@@ -0,0 +1,52 @@
+"use server";
```

Comment:
Could you, please, put this file and tests into `send-contact-form` folder in case if we need to declare new actions?

Reply:
extracted to an own folder